### ISSUE TITLE
Navigating the Event menu system

### ISSUE THREAD

**Issue by modulist** [iribarne/broward-drupal#65]
The objective of this UI is to create a mobile-friendly user experience with a completely flat menu structure and a layout based on a one-page design pattern.

Navigation will function like a mobile app's menu with a limited number of icons to lead to the main parts of the site. For an event site, the crucial links are 1) event information (venue, cost, dates, etc.); 2) schedule; 3) speakers; and 4) community. 

In order to pull this off, I'm proposing a new kind of menu that we can call an overflow menu. The menu will display only a few icon links with a **More...** button that leads to additional text links. 

## Normal state at 375 pixels wide:
![event menu - 375 normal](https://user-images.githubusercontent.com/713327/32676561-1944a9fc-c629-11e7-914e-22ed4d7fcb7b.png)

**Expanded state at 375 pixels wide**
When the **More...** link is expanded, you will see a search widget followed by a series of text links, and another **More...** link if the menu goes beyond a certain number of items.
![event menu - 375 expanded](https://user-images.githubusercontent.com/713327/32676647-6123cc12-c629-11e7-8eca-062405070e00.png)

The second **More...** link can be expanded to a menu with a vertical scrollbar if needed:
![popup - expanded](https://user-images.githubusercontent.com/713327/32676765-d4afebca-c629-11e7-913a-db8fd0487995.png)

## Expanded state at 768 pixels wide:
Here you can see a **Community** icon before the **More...** link, which was the first text link on the expanded menu in the 375 pixel version
![event menu - 768 expanded](https://user-images.githubusercontent.com/713327/32676778-e240f428-c629-11e7-90c2-f2dab72438cc.png)

<hr>

Additional specs for hover and active states can be found in [Figma ›](https://www.figma.com/file/guQlo64zsM5Bje4uWwTjIBiq/Relativity-icon-menus-1?node-id=15%3A348)

**Comment by shaal** [comment]
I think the second "more..." in the menu is confusing and results in a giant list of options,
Maybe instead it should say 'sitemap', and take users to a dedicated sitemap page?
https://www.nngroup.com/articles/site-map-usability/

**Comment by ghost** [comment]
I agree with David, the second more is confusing or maybe using an other word than "More..." and keep the same idea can help...
